broker-daemon/state-machines/fill-state-machine.js

```diff
@@ -313,6 +334,13 @@ const FillStateMachine = StateMachine.factory({
     onBeforeReject: function (lifecycle, error) {
       this.logger.error(`Encountered error during transition, rejecting`, error)
       this.fill.error = error
+    },
+    /**
+     * Returns true if there is a relayer error associated with the fill, false if not
+     * @return {Boolean}
+     */
+    isRelayerError: function () {
```

Comment:
were you able to use `get isRelayerError() {}` here or did it error on you?

Reply:
errored